File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = [None] * (2 * len(nums))
        for x in range(len(nums)):
            ans[x] = nums[x]
            ans[len(nums)+x] = nums[x]
        return ans

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int> productList;
        for (int x = 0; x < nums.size(); x++) {
            int product = 1;
            for (int y = 0; y < nums.size(); y++) {
                // if (nums[y] == 0) {
                //     product = 0;
                //     break;
                // }
                if(x != y) {
                    product *= nums[y];
                }
            }
            productList.push_back(product);
        }
        return productList;
    }
};
